Packs/ExodusIntelligence/IndicatorFields/indicatorfield-CPE_MATCH.json

```diff
@@ -0,0 +1,30 @@
+{
+    "associatedToAll": false,
+    "associatedTypes": [
+        "Exodus Intelligence"
+    ],
+    "caseInsensitive": true,
+    "cliName": "cpematch",
+    "closeForm": false,
+    "content": true,
+    "editForm": true,
+    "group": 2,
+    "hidden": false,
+    "id": "indicator_cpematch",
+    "isReadOnly": false,
+    "locked": false,
+    "name": "CPE MATCH",
+    "neverSetAsRequired": false,
+    "openEnded": false,
+    "ownerOnly": false,
+    "required": false,
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "longText",
```

Comment:
if its longtext its better to use markdown as it gives you better options with no real downside.